src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -325,6 +330,96 @@ create_reference_table(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * create_citus_local_table creates a citus table from the table with relationId.
+ * The created table would have the following properties:
+ *  - it will have only one shard,
+ *  - its distribution method will be DISTRIBUTE_BY_NONE,
+ *  - its replication model will be ReplicationModel,
+ *  - its replication factor will be set to 1.
+ * On the contrary of reference tables, a citus local table has only one placement
```

Comment:
this line of the comment is slightly confusing: `On the contrary of reference tables`

I get what you mean, but the sentence should be separated. Similar to ref.  tables, it has 1 placement, in addition to that the restriction is the placement is only allowed on the coordinator.